### PR TITLE
feat: prefinance project card

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: #2276

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

- See featured projects section on https://deploy-preview-2288--regen-marketplace.netlify.app/
- https://deploy-preview-2288--regen-marketplace.netlify.app/profiles/regen18xc64ltgtu7uthkpj4fug2dnm8z5m0narvy0ww/projects

Prefinance projects on the projects/ list page will be handled separately as part of #2277

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
> @erikalogie see testing instructions
@erikalogie ping

**Review by aaronc — APPROVED** [review]
LGTM pending @erikalogie's approval

**Comment by erikalogie** [comment]
@blushi so weird, I could have sworn I commented on this already, not sure what happened.

Here is my feedback: https://monosnap.com/file/juWW2dkZUSYFyhVx7NNYHS6LBjL4XV

**Pull request merged**
